RELEASE_NOTES.md

```diff
@@ -2,6 +2,10 @@
 
 * Adds metrics for cache refresh start and end times
 
+Niassa plugin:
+
+* Obliterate this plugin and any references to that nightmare.
```

Comment:
While I fully understand where this is coming from, can the language please be modified to something more neutral? `Obliterate this plugin and any references to that nightmare` -> `Obliterate this plugin and any references to it`

Reply:
Patches welcome.